www/js/modules/main/directives/histogramChart.js

```diff
@@ -131,13 +131,13 @@ export default ($log) => {
         controller: histogramController,
         controllerAs: 'histogramController',
         bindToController: {
-            dataSet: '@'
+            dataset: '='
```

Comment:
Also, to pass object references, we need to use two-way binding `=` as it's passed by reference. Otherwise you have to pass the value using `<directive data-dataset="{{ value }}">`, but you will only have a reference to the string value of the object.